Error if webhook is set v3.3.5

**v3.3.5** 

If I set any **webhook** (both in cli and as env var) I've got this in endless cycle:

```
created by main.main
	/src/cmd/git-sync/main.go:492 +0x1765
INFO: detected pid 1, running init handler
I0122 21:37:15.390135      12 main.go:454] "level"=0 "msg"="starting up" "pid"=12 "args"=["/git-sync","--webhook-url=https://home.cyberbrain.pw/api/webhook/reboot"]
I0122 21:37:15.390185      12 main.go:822] "level"=0 "msg"="cloning repo" "origin"="https://gitlab.cyberbrain.pw/infrastructure/deployment-sites/zimniy/home-assistant" "path"="/tmp/git"
panic: duplicate metrics collector registration attempted
goroutine 44 [running]:
github.com/prometheus/client_golang/prometheus.(*Registry).MustRegister(0xc0000be1e0, 0xc0000a0800, 0x1, 0x1)
	/src/vendor/github.com/prometheus/client_golang/prometheus/registry.go:391 +0xad
github.com/prometheus/client_golang/prometheus.MustRegister(...)
	/src/vendor/github.com/prometheus/client_golang/prometheus/registry.go:176
k8s.io/git-sync/pkg/hook.(*HookRunner).Run(0xc000182870, 0x9f0120, 0xc000026038)
	/src/pkg/hook/hook.go:109 +0x91
created by main.main
	/src/cmd/git-sync/main.go:492 +0x1765
INFO: detected pid 1, running init handler
I0122 21:37:15.822954      13 main.go:454] "level"=0 "msg"="starting up" "pid"=13 "args"=["/git-sync","--webhook-url=https://home.cyberbrain.pw/api/webhook/reboot"]
I0122 21:37:15.823014      13 main.go:822] "level"=0 "msg"="cloning repo" "origin"="https://gitlab.cyberbrain.pw/infrastructure/deployment-sites/zimniy/home-assistant" "path"="/tmp/git"
panic: duplicate metrics collector registration attempted
goroutine 50 [running]:
github.com/prometheus/client_golang/prometheus.(*Registry).MustRegister(0xc0000b61e0, 0xc00006f270, 0x1, 0x1)
	/src/vendor/github.com/prometheus/client_golang/prometheus/registry.go:391 +0xad
github.com/prometheus/client_golang/prometheus.MustRegister(...)
	/src/vendor/github.com/prometheus/client_golang/prometheus/registry.go:176
k8s.io/git-sync/pkg/hook.(*HookRunner).Run(0xc0003040c0, 0x9f0120, 0xc000026038)
	/src/pkg/hook/hook.go:109 +0x91
created by main.main
	/src/cmd/git-sync/main.go:492 +0x1765
```
Absolutely all except webhook works perfectly:
```
docker ps --format "{{.Names}}" | grep -q -w git-sync-home-assistant || \
docker run -d \
    --name git-sync-home-assistant \
    --restart always \
    --memory 8m \
    --cpu-shares 10 \
    -u$(id -u):$(id -g) \
    -e GIT_SYNC_REPO="https://.../home-assistant" \
    -e GIT_SYNC_BRANCH="production" \
    -e GIT_SYNC_USERNAME='gitlab+deploy-token-5' \
    -e GIT_SYNC_PASSWORD_FILE="/password" \
    -e GIT_SYNC_DEPTH="1" \
    -e GIT_SYNC_WAIT="30" \
    -e GIT_SYNC_HTTP_METRICS="false" \
    -v "${PWD}/password":/password:Z,ro \
    -v "${PWD}/config":/config:z,rw \
    registry.cyberbrain.pw/docker/git-sync-home-assistant:latest
```

--- a/pkg/hook/hook.go
+++ b/pkg/hook/hook.go
@@ -34,6 +34,10 @@ var (
 	}, []string{"name", "status"})
 )
 
+func init() {
+	prometheus.MustRegister(hookRunCount)
+}
+
 // Describes what a Hook needs to implement, run by HookRunner
 type Hook interface {
 	// Describes hook
@@ -115,7 +119,6 @@ func (r *HookRunner) Send(hash string) {
 // Run waits for trigger events from the channel, and run hook when triggered
 func (r *HookRunner) Run(ctx context.Context) {
 	var lastHash string
-	prometheus.MustRegister(hookRunCount)
 
 	// Wait for trigger from hookData.Send
 	for range r.data.events() {

